pkg/stages/steps_install.go

```diff
@@ -503,7 +695,8 @@ func GetInstallProviderBinaries(sis values.System, l types.KairosLogger) error {
 
 	// Link /system/providers/agent-provider-kairos to /usr/bin/kairos, not sure what uses it?
 	// TODO: Check if this is needed, maybe we can remove it?
-	err = os.Symlink("/system/providers/agent-provider-kairos", "/usr/bin/kairos")
+	providerPath := fmt.Sprintf("/system/providers/agent-provider-%s", providerBinaryName)
+	err = os.Symlink(providerPath, "/usr/bin/kairos")
```

Comment:
Here also related to the distinction between the two types of providers do the non kairos provider need to also be linked to `/usr/bin/kairos`?